salt/utils/vmware.py

```diff
@@ -615,7 +615,7 @@ def get_content(service_instance, obj_type, property_list=None,
     # Create object spec to navigate content
     obj_spec = vmodl.query.PropertyCollector.ObjectSpec(
         obj=obj_ref,
-        skip=False,
+        skip=True,
```

Comment:
@cro This would break the search for local properties in the container. Can you please update the code to be `False if local_properties else True`, so the code still works for local properties. Thanks!